relentless/optimize/objective.py

```diff
@@ -6,7 +6,7 @@
 by adjusting the variables on which the function depends.
 
 This function, :math:`f`, is a scalar value that is defined as a function of :math:`n`
-problem :class:`DesignVariables<~relentless.variable.DesignVariable>`
+problem :class:`DesignVariables<~relentless.variable.DesignVariable>`                           #this doesn't make a hyperlink, do you know how to fix this?
```

Comment:
I'll need to investigate. I thought this pattern worked when I tried it before... the goal is to get it to show `DesignVariables` (plural) but link to `DesignVariable` (singular). If nothing else, you can replace it with a regular reference to `DesignVariable` and we can ignore the pluralization.

Reply:
I can't figure out how to get this to work, so convert to:
```suggestion
problem :class:`~relentless.variable.DesignVariable>`\s
```
which will print the link with an s right after it.